src/components/core/SideNav/SideNav.js

```diff
@@ -87,16 +89,15 @@ const SideNav = ({
   const [activePath, setActivePath] = useState(initialPath);
   const toggleState = (path) => () => setActivePath(path);
 
-  const categoryMeta = useCategoryMeta(category);
-
-  useScrollSpy(".heading", (url) => url && setActivePath(getArticlePath(url)));
+  // const categoryMeta = useCategoryMeta(category);
+  // useScrollSpy(".heading", (url) => url && setActivePath(getArticlePath(url)));
 
   const navColor = getVersionColor(selectedVersion, versions);
 
   return (
     <Nav color={navColor} expanded={expanded} setExpanded={setExpanded}>
       <NavHeader>
-        <Link to={"/"} style={{ color: "inherit" }}>
+        <Link href="/" style={{ color: "inherit" }}>
```

Comment:
It's missing `<a>` tag here. Look at documentation -> https://nextjs.org/docs/migrating/from-gatsby#links